src/send-email.js

```diff
@@ -50,5 +67,135 @@ async function fetchFilesFromRepo() {
+    }
+
+    if (problems.length > 0) {
+      filtered.push({ ...project, problems: problems });
+    }
+  });
+  return filtered;
+}
+
+export function filterNew(projects, time){ // projects is an array of objects; time is the number of days to search back from day of email
+  const filtered = projects.filter(project => parse(project.created) < subDays(parse(project.created), time));
+  return filtered;
+}
+export function formatEmail(projects, greeting){ // filtered list of projects, json
+  let dom = new JSDOM("<!DOCTYPE html>"),
+      document = dom.window.document;
+  const site = process.env.site;
+  const greetingDiv = document.createElement("div");
+  addElement("Hello,", greetingDiv);
+  greetingDiv.append(document.createElement("br"));
+  addElement(greeting, greetingDiv);
+  document.body.append(greetingDiv);
+
+  const projectDiv = document.createElement("div");
+  addElement("<h2>Projects</h2>", projectDiv);
+  if (projects.length == 0){
+    addElement("No projects found", projectDiv);
+  }
+  projects.forEach((project) => {
+    addElement(`Project Title: ${project.title}`, projectDiv);
+    addElement(`Contact Name: ${project.mainContact.name}`, projectDiv);
+    addElement(`Contact Email: ${project.mainContact.email}`, projectDiv);
+    addElement(`URL: ${site}/${project.slug}`, projectDiv);
+    addElement(`Possible Problems: ${project.problems.join(" ")}`, projectDiv);
+    projectDiv.append(document.createElement("br"));
+  });
+  document.body.append(projectDiv);
+
+  const endingDiv = document.createElement("div");
+  document.body.append(endingDiv);
+
+  console.debug(dom.serialize());
+  return dom.serialize();
+}
+
+export async function sendNodeMail(to, subject, body){
+
+  // This smtp was set up by Brown OIT unix team -- this will only work on Brown internal network (such as BKE)
+  // Auth not needed at this time
+  let EMAIL_SMTP = "smtp://mail-relay.brown.edu:25";
+  let transporter;
+  // Initiate transporter
+  if (EMAIL_SMTP !== undefined) {
+    transporter = nodemailer.createTransport(EMAIL_SMTP);
+  } else {
+    let testAccount = await nodemailer.createTestAccount();
+    transporter = nodemailer.createTransport({
+      host: "smtp.ethereal.email",
+      port: 587,
+      secure: false, // true for 465, false for other ports
+      auth: {
+        user: testAccount.user, // generated ethereal user
+        pass: testAccount.pass, // generated ethereal password
+      },
+    });
+  }
+
+  let info = await transporter.sendMail({
+    from: '"Brown Policy Lab" <no-reply@brown.edu>', // sender address
+    to: to, // list of receivers
+    subject: subject, // Subject line
+    html: body, // html body
+  });
+
+  console.debug("Message sent: %s", info.messageId);
+  // Message sent: <b658f8ca-6296-ccf4-8306-87d57a0b4321@example.com>
+
+}
+
+function addElement(text, div) {
+
+  const newContent = document.createElement("p");
+  newContent.innerHTML = text;
+  div.appendChild(newContent);
+}
```

Comment:
maybe shift this function up to line 158, so that it's right next to the email formatting bit?